lib/access-context.js

```diff
@@ -32,6 +32,7 @@ function AccessContext(context) {
   }
   context = context || {};
 
+  this.registry = context.registry;
   this.principals = context.principals || [];
   var model = context.model;
   model = ('string' === typeof model) ? loopback.getModel(model) : model;
```

Comment:
saw that comment lately : yes i changed for
`model = ('string' === typeof model) ? this.registry.getModel(model) : model;`